iceoryx_posh/source/roudi/roudi.cpp

```diff
@@ -212,13 +212,26 @@ void RouDiMultiProcess::processMessage(const runtime::MqMessage& message,
             capro::ServiceDescription service(cxx::Serialization(message.getElementAtIndex(2)));
             cxx::Serialization portConfigInfoSerialization(message.getElementAtIndex(4));
 
-            m_prcMgr.addReceiverForProcess(processName,
+            m_prcMgr.addReceiverForProcess(cxx::CString100(cxx::TruncateToCapacity, processName),
                                            service,
-                                           message.getElementAtIndex(3),
+                                           cxx::CString100(cxx::TruncateToCapacity, message.getElementAtIndex(3)),
                                            iox::runtime::PortConfigInfo(portConfigInfoSerialization));
         }
         break;
     }
+    case runtime::MqMessageType::IMPL_CONDITION_VARIABLE:
+    {
+        if (message.getNumberOfElements() != 2)
+        {
+            LogError() << "Wrong number of parameter for \"MqMessageType::IMPL_CONDITION_VARIABLE\" from \""
```

Comment:
Log or MODERATE error?

Reply:
I'd propose to first define a concept with #130 and then adapt the code base accordingly.

Reply:
Yes, I agree.